packages/path-compare/test/index.test.js

```diff
@@ -0,0 +1,58 @@
+import test from '@endo/ses-ava/prepare-endo.js';
+import { pathCompare, stringCompare } from '../src/index.js';
+
+test('stringCompare - returns 0 for equal strings', t => {
+  t.is(stringCompare('abc', 'abc'), 0);
+});
+
+test('stringCompare - returns negative for a < b', t => {
+  t.true(stringCompare('abc', 'def') < 0);
+});
+
+test('stringCompare - returns positive for a > b', t => {
+  t.true(stringCompare('def', 'abc') > 0);
+});
+
+test('pathCompare - returns 0 for equal arrays', t => {
+  t.is(pathCompare(['a', 'b'], ['a', 'b']), 0);
+});
+
+test('pathCompare - returns negative for shorter array', t => {
+  t.true(pathCompare(['a'], ['a', 'b']) < 0);
+});
+
+test('pathCompare - returns positive for longer array', t => {
+  t.true(pathCompare(['a', 'b'], ['a']) > 0);
+});
+
+test('pathCompare - returns negative for smaller cumulative length', t => {
+  t.true(pathCompare(['a', 'b'], ['aa', 'bb']) < 0);
+});
+
+test('pathCompare - returns positive for larger cumulative length', t => {
+  t.true(pathCompare(['aa', 'bb'], ['a', 'b']) > 0);
+});
```

Comment:
I'd like these tests to cover smaller/larger cumulative length where the naïve element-by-element approach would favor the opposite result, e.g. `["bb", "aa"]` before `["a", "bbbb"]`.